src/MetricsLogger.tsx

```diff
@@ -2,20 +2,22 @@ import { FC, useEffect } from 'react';
 
 import { ObserveCallback, useMetrics } from './MetricsContext';
 
-const handleObserve: ObserveCallback = ({ metricName, value, tags }) => {
+const defaultLogger: ObserveCallback = ({ metricName, value, tags }) => {
   // eslint-disable-next-line no-console
   console.log('[prom_react]', metricName, value, tags);
 };
 
-const MetricsLogger: FC = () => {
+const MetricsLogger: FC<{ logger?: ObserveCallback }> = ({
+  logger = defaultLogger,
+}) => {
   const { addObserveListener, removeObserveListener } = useMetrics();
 
   useEffect(() => {
-    addObserveListener(handleObserve);
+    addObserveListener(logger);
     return () => {
-      removeObserveListener(handleObserve);
+      removeObserveListener(logger);
     };
-  }, [addObserveListener, removeObserveListener]);
+  }, [addObserveListener, logger, removeObserveListener]);
```

Comment:
Should we encourage to memoize `logger` function to prevent observe listener from being removed/added on each render?

Reply:
Yep you're completely right. I improved the docs with your suggestion. Can you take a look please?